compair/static/modules/course/course-duplicate-partial.html

```diff
@@ -191,7 +191,85 @@ <h3 class="media-heading">
+                                            is-open="assignment.date.sestart.opened" ng-required="assignment.enable_self_evaluation && assignment.selfEvalCheck"
+                                            datepicker-options="{ minDate: datePickerMinDate(assignment.date.sestart.date, duplicateCourse.date.course_start.date), maxDate: datePickerMaxDate(assignment.date.seend.date, duplicateCourse.date.course_end.date) }"
+                                            />
+                                        <span class="input-group-btn">
+                                            <button type="button" class="btn btn-default" ng-click="datePickerOpen($event, assignment.date.sestart)"><i class="glyphicon glyphicon-calendar"></i></button>
+                                        </span>
+                                    </span>
+                                </div>
+                                <div class="assignment-date">
+                                    <div uib-timepicker ng-model="assignment.date.sestart.time" minute-step="1" mousewheel="false"></div>
+                                </div>
+                            </div>
+                            <div class="col-md-6 form-group">
+                                <label class="required-star">Self-Evaluation Ends</label><br />
+                                <div class="assignment-date">
+                                    <span class="input-group">
+                                        <input type="text" class="form-control" uib-datepicker-popup="{{format}}" ng-model="assignment.date.seend.date"
+                                            is-open="assignment.date.seend.opened" ng-required="assignment.enable_self_evaluation && assignment.selfEvalCheck"
+                                            datepicker-options="{ minDate: datePickerMinDate(assignment.date.sestart.date, assignment.date.sestart.date, duplicateCourse.date.course_start.date), maxDate: datePickerMaxDate(duplicateCourse.date.course_end.date) }"
+                                            />
+                                        <span class="input-group-btn">
+                                            <button type="button" class="btn btn-default" ng-click="datePickerOpen($event, assignment.date.seend)"><i class="glyphicon glyphicon-calendar"></i></button>
+                                        </span>
+                                    </span>
+                                </div>
+                                <div class="assignment-date">
+                                    <div uib-timepicker ng-model="assignment.date.seend.time" minute-step="1" mousewheel="false"></div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="row" ng-show="assignment.enable_self_evaluation">
```

Comment:
I don't think we need `self_eval_instructions` in the duplication form. We're just trying to quickly duplicate multiple assignments so we reduced the form to just the dates which are very important and need to change.